docusaurus/sidebars.js

```diff
@@ -17,11 +17,7 @@ module.exports = {
           {
             type: 'category',
             label: 'Analytics',
-            items: [
-              'components/analytics/index',
-              'components/analytics/analytics',
-              'components/analytics/hook',
-            ],
+            items: ['components/analytics/index', 'components/analytics/analytics', 'components/analytics/hook'],
```

Comment:
back to single lines?

Reply:
Last week I updated the prettier settings to use 120 as the column width instead of 80. This may be from those changes